docs/projects.md

```diff
@@ -2,7 +2,23 @@
 
 Here you find an overview of our [current](#current-projects) and [completed projects](#completed-projects). According to Open Science principles, most of our software is publicly accessible on GitHub, and can be found via the title link.
 
+
 ### Current projects
+| Mental-health-recovery                                                                                                                                                                                       |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+| **Time frame**: 07/2024 - present                                                                                                                                                                            |
+| **Research domain**: ODISSEI Social Data Science Team (SoDa)                                                                                                                                                 |
```

Comment:
For homogeneity, I would put the Faculty of Social and Behavioural Sciences as the research domain. Then, you can mention the SoDa team in the description. Perhaps at some point we should extend the template to include researchers/collaborators.

Reply:
thanks for the comment. Done!